pilot/pkg/proxy/envoy/v2/ads.go

```diff
@@ -522,47 +530,59 @@ func (s *DiscoveryServer) removeCon(conID string, con *XdsConnection) {
 
 // getServicesForEndpoint returns the list of services associated with a node.
 // Currently using the node endpoint IP.
-func (s *DiscoveryServer) getServicesForEndpoint(node *model.Proxy) ([]*model.ServiceInstance, error) {
-	// TODO: cache the results, this is a pretty slow operation and called few times per
-	// push
-	proxyInstances, err := s.env.GetProxyServiceInstances(node)
-	return proxyInstances, err
-}
+//func (s *DiscoveryServer) getServicesForEndpoint(node *model.Proxy) ([]*model.ServiceInstance, error) {
+//	// TODO: cache the results, this is a pretty slow operation and called few times per
+//	// push
+//	proxyInstances, err := s.env.GetProxyServiceInstances(node)
+//	return proxyInstances, err
+//}
```

Comment:
why aren't we just removing this?

Reply:
Coz we are going to enable this soon.. thought i would keep the code around